Formula/parliament.rb

```diff
@@ -0,0 +1,72 @@
+class Parliament < Formula
+  include Language::Python::Virtualenv
+
+  desc "AWS IAM linting library"
+  homepage "https://github.com/duo-labs/parliament"
+  url "https://github.com/duo-labs/parliament/archive/1.3.1.tar.gz"
```

Comment:
```suggestion
  url "https://files.pythonhosted.org/packages/7e/82/26b58070877848c21c59a7c8325117941ec136552425212daa0968e8afcd/parliament-1.3.1.tar.gz"
```
If you change to the PyPI sdist url, `brew bump-formula-pr` will know to update the python resources on a version bump. You can still refer to the source repo by putting 'https://github.com/duo-labs/parliament.git' in the `head` section of the formula.

Reply:
Thanks @dtrodrigues, will do!